src/components/LeftSidebar/LeftSidebar.vue

```diff
@@ -78,14 +79,26 @@
 			<!-- New Conversation -->
 			<NewGroupConversation v-if="canStartConversations"
 				ref="newGroupConversation"
-				class="new-conversation__button" />
+				class="new-conversation__button"
+				:class="{'hidden-visually': isFocused}" />
```

Comment:
It will conflict with open conversations, so as I said earlier, let's wait for it to be merged

Reply:
yes, I will merge it after the PR of open conversations.